### DISCUSSION TITLE
How to edit document author info?

### DISCUSSION THREAD

**Discussion by l3land** [mush42/bookworm#178] [Q&A]
Hi there,
New Bookworm user (and Windows user at that) hoping this is the right forum to post a question like this. Wondering how to edit the book info, such as author. I can see an option to edit the book's title with f2 or context menu, but when I open the document info dialog, none of the fields are editable!
Thanks,
Andrew

**Comment by Cambridgeport90** [comment]
Is this an author that was imported as unknown, perhaps? I have seen that occur quite a lot when I import stuff to my library; not sure if it’s missing metadata in the Epub files.

From: l3land ***@***.***>
Sent: Monday, July 25, 2022 11:12 AM
To: blindpandas/bookworm ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [blindpandas/bookworm] How to edit document author info? (Discussion #178)


Hi there,
New Bookworm user (and Windows user at that) hoping this is the right forum to post a question like this. Wondering how to edit the book info, such as author. I can see an option to edit the book's title with f2 or context menu, but when I open the document info dialog, none of the fields are editable!
Thanks,
Andrew

**Reply by l3land** [reply]
Thanks @Cambridgeport90  
That's right-- files I download from bookshare often missing essential metadata like author. Tried to add in calibre but it created a separate metadata file that bookworm doesn't recognize? Can anyone help me modify spin metadata so that I can update missing author info in bookworm? Thanks!
